draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -302,7 +302,7 @@ DIGIT = %x30-39 ; 0-9
 BASE64URL = 1*(ALPHA / DIGIT / "-" / "_")
 JWT = BASE64URL "." BASE64URL "." BASE64URL
 DISCLOSURE = BASE64URL
-SD-JWT = JWT "~" *[DISCLOSURE "~"]
+SD-JWT = JWT "~" *(DISCLOSURE "~")
```

Comment:
Thank you.
This simplification avoids problems with tools that validate ABNF via a PEG parser.
It also avoids a double take by people trying to read the grammar.

Reply:
someday I hope to not do a double take when mixing up @c2bo and @cabo